learnArrays/learnArrays.js

```diff
@@ -0,0 +1,36 @@
+// function to find maximum number in the array
+// Args : array of numbers
+
+export let findMax = (testArr) => { // [2,3,4]
+    if (!Array.isArray(testArr)) {
+        return "Please Pass Array"
+    }
+    let maxNum = Number.NEGATIVE_INFINITY
+    testArr.forEach(element => {
+        if (element > maxNum) {
+            maxNum = element
+        }
+    });
+    return maxNum;
+}
+
+export let findCommonEle = (firstArr, secondArr) => {
```

Comment:
Change name of the function. There is one more function with the name `findCommonElem`

Reply:
fixed